lte/gateway/python/magma/enodebd/tests/BUILD.bazel

```diff
@@ -61,11 +61,19 @@ pytest_test(
     deps = [
         "//dp/protos:enodebd_dp_python_proto",
         "//lte/gateway/python/magma/enodebd/data_models:data_model",
+        "//lte/gateway/python/magma/enodebd/data_models:data_model_parameters",
```

Comment:
This dependency is not needed. Also not needed: line 65, 67, 70, 74, 75.
(FYI: 76 is needed, because parameterized was added in the test, 69 can stay as it is actually imported in the test, but is not essential because it was pulled as a transitive dependency before.)

Reply:
Removed unnecessary dependencies from Bazel file.